### PR TITLE
New cav/ia gsim from Sandikkaya and Akkar 2017

### PR DESCRIPTION
**Pull request by kslytherin**
The pull request implements the Sandikkaya et al. (2017) GMPE for Arias Intensity and CAV. We have compared the mean and standard deviations to other models for CAV and AI and it gives reasonably similar results. This implementation has been verified with test data for the predicted mean values provided by the developers of the model.

### CODE REVIEW

**Comment by emthompson-usgs** [comment]
@micheles Can you give us any guidance on what happened with the docs check? I can't make sense of the log.

**Comment by raoanirudh** [comment]
> Can you give us any guidance on what happened with the docs check? I can't make sense of the log.

Probably related to #9512, introduced to ensure new contributors are also acknowledged in [CONTRIBUTORS.txt](https://github.com/gem/oq-engine/blob/master/CONTRIBUTORS.txt)!

File: openquake/hazardlib/gsim/sandikkaya_akkar_2017.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+    #: Required rupture parameters are
+    REQUIRES_RUPTURE_PARAMETERS = {"mag", "rake"}
+
+    #: Required distance measures are
+    REQUIRES_DISTANCES = {"rjb"}
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        [self.kind] = self.REQUIRES_DISTANCES
```
Comment (micheles) [inline openquake/hazardlib/gsim/sandikkaya_akkar_2017.py:111]: You can remove the `__init__` method entirely and just set at class level `kind = "rjb"`

File: openquake/hazardlib/gsim/sandikkaya_akkar_2017.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+        for spec of input and result values.
+        """
+
+        # Get mean and standard deviation
+        for m, imt in enumerate(imts):
+            conv_fact = 0
+            conv_fact2 = 0
+            ofact = 100
+            gfact = 9.81
+            if str(imt) == "CAV":
```
Comment (micheles) [inline openquake/hazardlib/gsim/sandikkaya_akkar_2017.py:126]: I would use `imt.string == "CAV"".

File: openquake/hazardlib/gsim/sandikkaya_akkar_2017.py [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+import numpy as np
+
+from openquake.baselib.general import CallableDict
+from openquake.hazardlib.gsim.base import GMPE, CoeffsTable
+from openquake.hazardlib import const
+from openquake.hazardlib.imt import CAV, IA
+
+
+_compute_logarithmic_distance_term = CallableDict()
+
```
Comment (micheles) [inline openquake/hazardlib/gsim/sandikkaya_akkar_2017.py:33]: If I remember correctly, we don't use underscores for similar CallableDicts in other parts of hazardlib, I would just call it `compute_logarithmic_distance_term`.

File: openquake/hazardlib/gsim/sandikkaya_akkar_2017.py (hunk 2 above)
Comment (kslytherin) [reply openquake/hazardlib/gsim/sandikkaya_akkar_2017.py:126]: this worked. thanks!

File: openquake/hazardlib/gsim/sandikkaya_akkar_2017.py (hunk 3 above)
Comment (kslytherin) [reply openquake/hazardlib/gsim/sandikkaya_akkar_2017.py:33]: this worked. thanks!

**Review by mmpagani — APPROVED** [review]
Many thanks!

**Pull request merged**